index.html

```diff
@@ -7,6 +7,6 @@
     <title> Welcome to Potionfy</title>
 </head>
 <body>
-
+	<h1>Welcome to Potionfy! enjoy your stay.</h1>
```

Comment:
Could you remove the "Enjoy your stay" string and make the "Welcome to Potionfy!" string red?